pkg/render/logstorage.go

```diff
@@ -207,6 +207,7 @@ type ElasticsearchConfiguration struct {
 	KbService                   *corev1.Service
 	ClusterDomain               string
 	DexCfg                      DexRelyingPartyConfig
+	ManagerDomain               string // Domain where the manager is reachable, for setting Kibana publicBaseUrl
```

Comment:
If this variable would be called KibanaPublicURL, you would need no comment to clarify it and `fmt.Sprintf("%s/%s", es.cfg.ManagerDomain, KibanaBasePath)`  could then be done in the controller, saving you an if-statement on line 1263. WDYT?

Reply:
I like the idea but I think it is more appropriate to construct it in the render, that is the correct scope for doing that.